### PR TITLE
refactor(typings): export as es2015 module

### PR DESCRIPTION
**Pull request by MeirionHughes**
> dep: https://github.com/Level/abstract-leveldown/pull/129

reworked as es2015 module (default export). ⚠️ publish as minor;    

image

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,31 +1,22 @@
...
+  new <K=any, V=any, O={}, PO={}, GO={}, DO={}, IO={}, BO={}>(
+    db: AbstractLevelDOWN<any, any, O, PO, GO, DO, IO, BO>,
+    options?: CodecOptions
+  ): EncodingDOWN<K, V, O, PO, GO, DO, IO, BO>
+}
 
-export = encoding;
\ No newline at end of file
+declare const EncodingDOWN: EncodingDOWNConstructor;
+export default EncodingDOWN;
```
Comment (zixia) [inline index.d.ts]: I'd like to suggest to also add an `export { EncodingDOWN }` at the end of file.

Comment (MeirionHughes) [reply index.d.ts]: this one is already exported too: 

image

**Comment by MeirionHughes** [comment]
@ralphtheninja good to go as patch.

**Comment by ralphtheninja** [comment]
2.3.1

**Pull request merged**